### PR TITLE
Add runAsNonRoot to false for initChown

### PR DESCRIPTION
**Pull request by StrongMonkey**
Specifically set runAsNonRoot to false for initChown container so that
it can always run as root. This is to avoid any pod.securityContext overriding
runAsNonRoot.

### CODE REVIEW

**Comment by StrongMonkey** [comment]
Related issue: https://github.com/rancher/rancher/issues/29639

File: charts/grafana/templates/_pod.tpl [hunk 1]
```diff
@@ -27,6 +27,7 @@ initContainers:
     {{- end }}
     imagePullPolicy: {{ .Values.initChownData.image.pullPolicy }}
     securityContext:
+      runAsNonRoot: false
       runAsUser: 0
```
Comment (cbron) [inline charts/grafana/templates/_pod.tpl:31]: Weird, wouldn't `runAsUser` handle this for you ? Or do you need both to enable root user.

**Comment by zanhsieh** [comment]
@StrongMonkey 
You need to bump the chart version to latest. Without it, this PR can't merge.

**Comment by StrongMonkey** [comment]
@zanhsieh Chart version is updated.

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**